src/cli/services/linter/utils/listFiles.ts

```diff
@@ -1,10 +1,12 @@
 import { normalize } from '@stoplight/path';
 import * as fg from 'fast-glob';
 
-export async function listFiles(pattens: string[]): Promise<string[]> {
-  const { files, urls } = pattens.reduce<{ files: string[]; urls: string[] }>(
+export async function listFiles(pattens: Array<number | string>): Promise<Array<number | string>> {
+  const { files, fds, urls } = pattens.reduce<{ files: string[]; urls: string[]; fds: number[] }>(
```

Comment:
Could we rename fds to fileDescriptors or something? I'm not sure what a file descriptor is but that's more readable than fds.